packages/repmgr13/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Do we need the Dockerfile environment + build script?  Will the package build in mock?
```
mock --spec=./repmgr.spec -r centos-stream-8-x86_64 --sources=./
---OR---
mock --spec=./repmgr.spec -r centos-stream+epel-8-x86_64 --sources=./
```

Reply:
I don't know. I didn't need it before when I built it against ubi8 with centos 8 repositories.  🤷

Reply:
With https://github.com/ManageIQ/manageiq-rpm_build/pull/309#pullrequestreview-1160102936 and https://github.com/ManageIQ/manageiq-rpm_build/pull/309#pullrequestreview-1160098422 it will build in mock with `mock --spec=./repmgr.spec -r centos-stream+epel-8-x86_64 --sources=./` so we shouldn't need a Dockerfile and build script.

Reply:
Right.  I don't have mock so the Dockerfile is what I used for building it.